discover/commerce/articles/01-catalog-management/01-product-categories.markdown

```diff
@@ -0,0 +1,112 @@
+# Product Categories [](id=product-categories)
+
+Products can be organized into *Categories*. These groupings allow multiple
+products to be handled as a single unit. For example, a product category could
+be used to apply discounts or other offers to a set of products, to assist
+buyers in finding products, or to promote certain products to a specified user
+segment.
+
+To organize products into categories, you must create the categories first and
+then assign products to them.
+
+## Creating Categories [](id=creating-categories)
+
+To create vocabularies, categories and subcategories, go to *Site Menu* &rarr;
+*Content* &rarr; *Categories*. To learn more about the process, go to the
+[Defining Categories for
+Content][/discover/portal/-/knowledge_base/7-1/organizing-content-with-tags-and-categories#defining-categories-for-content]
+section.
+
++$$$
+
+Note: Categories are designed to be nested into a hierarchy. While this allows
+multiple categories to be grouped together and handled as a single unit, it
+also makes it difficult for administrators to see the entire vocabulary. To
+view all categories at once, go to *Control Panel* &rarr; *Configuration*
+&rarr; *System Settings* &rarr; *Assets* and select *Flattened Tree* from the
+drop-down menu in *Asset Categories Web*. Then return to *Site Menu* &rarr;
+*Content* &rarr; *Categories*.
+
+$$$
+
+## Adding and Removing Products from Categories [](id=adding-and-removing-products-from-categories)
+
+To add a product to one or more categories:
```

Comment:
A colon can only be used at the end of an independent clause (i.e., a sentence).